WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/PostsListPage.kt

```diff
@@ -20,17 +20,10 @@ object PostsListPage {
 
     // Workaround for cases when recyclerview id missing
     private val recyclerView: RecyclerView?
-        private get() {
+        get() {
             val pager = WPSupportUtils.getCurrentActivity().findViewById<ViewPager>(R.id.postPager)
             var recyclerView = pager.getChildAt(pager.currentItem)
                 .findViewById<View>(R.id.recycler_view) as RecyclerView
-            if (recyclerView == null) {
-                // Workaround for cases when recyclerview id missing
-                recyclerView = ((pager.getChildAt(pager.currentItem)
-                    .findViewById<View>(R.id.ptr_layout) as ViewGroup).getChildAt(0) as ViewGroup).getChildAt(
-                    0
-                ) as RecyclerView
-            }
```

Comment:
I was a bit confused by this removal, and then saw that it was caused by having a `Condition 'recyclerView == null' is always 'false'` warning otherwise. It looks like the value of `recyclerView` could be `null` in times of Java, but it'll never be `null` in Kotlin? 🤔 

Regardless of this, `PostsListPage` is only used in `WPScreenshotTest` in `editBlogPost` function, which is not used, so I guess we're fine with this adjustment 🤷